opencti-platform/opencti-graphql/config/test.json

```diff
@@ -99,6 +99,9 @@
     "enabled": true,
     "interval": 6000000
   },
+  "ingestion_manager": {
+    "enabled": false
```

Comment:
Because for testing pupose it's better to call it once, than wait until manager runs. I think it will avoid flakes.